src/Notification/Notification.js

```diff
@@ -0,0 +1,351 @@
+require('moonstone');
+
+/**
+* Contains the declaration for the {@link module:moonstone/Notification~Notification} kind.
+* @module moonstone/Notification
+*/
+
+var
+	kind = require('enyo/kind'),
+	Control = require('enyo/Control'),
+	dispatcher = require('enyo/dispatcher'),
+	EnyoHistory = require('enyo/History'),
+	Popup = require('enyo/Popup'),
+	ShowingTransitionSupport = require('enyo/ShowingTransitionSupport');
+
+var
+	Spotlight = require('spotlight');
+
+var
+	Button = require('../Button'),
+	BodyText = require('../BodyText'),
+	HistorySupport = require('../HistorySupport');
+
+/**
+* {@link module:moonstone/Notification~Notification} is a toast-like minimal popup that comes up
+* from the bottom of the screen. It requires a button to be provided and present to close it.
+*
+* @class Notification
+* @extends module:enyo/Popup~Popup
+* @ui
+* @public
+*/
+module.exports = kind(
+	/** @lends module:moonstone/Notification~Notification.prototype */ {
+
+	/**
+	* @private
+	*/
+	name: 'moon.Notification',
+
+	/**
+	* @private
+	*/
+	kind: Popup,
+
+	/**
+	* @private
+	*/
+	mixins: [HistorySupport, ShowingTransitionSupport],
+
+	/**
+	* @private
+	*/
+	showing: false,
+
+	/**
+	* @private
+	*/
+	classes: 'moon moon-neutral enyo-unselectable moon-notification',
+
+	/**
+	* @private
+	*/
+	floating: true,
+
+	/**
+	* @private
+	*/
+	spotlight: 'container',
+
+	/**
+	* @private
+	*/
+	modal: true,
+
+	/**
+	* @private
+	*/
+	spotlightRememberFocus: false,
+
+	/**
+	* @private
+	*/
+	activator: null,
+
+	/**
+	* @private
+	*/
+	shownMethod: 'shown',
+
+	/**
+	* @private
+	*/
+	hiddenMethod: 'hidden',
+
+	/**
+	* @private
+	*/
+	handlers: {
+		onRequestScrollIntoView   : '_preventEventBubble'
+	},
+
+	/**
+	* @private
+	* @lends module:moonstone/Popup~Popup.prototype
```

Comment:
I was initially under the impression we were deprecating `published` but saw no evidence of it, outside my own work. I used it here because it seemed like nobody was deeply interested in not having it any more. I can take it out, but it will just make this control look different from others. Thoughts?